index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio Page</title>
+</head>
+<body>
+  <main>
+    <header>
+      <nav class="flexr container">
+        <a href="#" class="logo">
```

Comment:
- [ ] as demonstrated by the previous reviewer, the logo is not inside the nav tag. please kindly follow that for semantic reasons.
- [ ] please centralize your headline heading and text so it matches the design. 
- [ ] kindly add some padding between the lets connect, the text above it and the social links below it. 
yours
![Screenshot (1056)_LI](https://user-images.githubusercontent.com/65298048/236399403-69f92428-4665-4d4d-8987-5d344fe5fafe.jpg)
original
![Screenshot (1057)](https://user-images.githubusercontent.com/65298048/236399446-8e8bc27e-53b0-4454-ba2d-9fee2103f36a.png)